src/lib/strings/time.ts

```diff
@@ -15,17 +17,17 @@ export function ago(date: number | string | Date): string {
   }
   const diffSeconds = Math.floor((Date.now() - ts) / 1e3)
   if (diffSeconds < NOW) {
-    return `now`
+    return t`now`
   } else if (diffSeconds < MINUTE) {
-    return `${diffSeconds}s`
+    return t`${diffSeconds}s`
```

Comment:
Is there another short hand way of saying "seconds" in Korean? For example, this text could be `10s`. Is there a reason to substitute the `s` in Korean?

Reply:
In Korea, time is typically written in Hangul, so s is replaced with "초", m with "분", and h with "시간". This will also change in Japanese and Chinese to their own language.

Reply:
In CJK, `10s` is not usally. Replacing time markers like `s` in CJK with your own language is the same for most websites, including Twitter, Instagram, and Facebook.

Reply:
I'll let @pfrazee take the call here